key_manager/hd_key/Cargo.toml

```diff
@@ -27,4 +27,4 @@ secp256k1 = { version = "0.27.0", features = ["rand-std"]}
 sha2 = { version = "0.10.6", default-features = false }
 slip44 = "0.1.4"
 thiserror = "1.0.40"
-zeroize = { version = "1.6.0", features = ["zeroize_derive"] }
+zeroize = { version = "1.3.0", features = ["zeroize_derive"] }
```

Comment:
zeroize can be removed as it was removed in the main branch

Reply:
For some reason, zeroize is still present in hd_key on main. I've opened an issue to remove it, which I'll attend to immediately following this PR being merged: #117